uniffi/src/bindings/kotlin/templates/RustBufferHelpers.kt

```diff
@@ -141,6 +141,36 @@ fun Double.lowerInto(buf: ByteBuffer) {
     buf.putDouble(this)
 }
 
+fun String.lower(): String {
+    return this
+}
+
+fun String.lowerInto(buf: ByteBuffer) {
+    buf.putInt(this.length)
+    buf.put(this.toByteArray())
+}
+
+fun String.Companion.liftFrom(buf: ByteBuffer): String {
+    val len = Int.liftFrom(buf)
+    val byteArr = ByteArray(len)
+    buf.get(byteArr)
+    return byteArr.toString(Charsets.UTF_8)
+}
+
+fun String.lowersIntoSize(): Int {
+    return this.length + 4
```

Comment:
This needs to be the number of utf8 bytes, not the number of characters; I don't know enough about unicode strings in Kotlin to know whether this is correct or not.